### PR TITLE
Updated SOAS hyperlink and added Trisquel

### PR DESCRIPTION
**Pull request by S-kus**
Fixes: #342 

Updated the link in SOAS from [installation page](https://wiki.sugarlabs.org/go/Sugar_on_a_Stick/Installation) to [this](https://wiki.sugarlabs.org/go/Sugar_on_a_Stick).

Added a hypertext for Trisquel sugar which leading [here](https://wiki.sugarlabs.org/go/Trisquel_On_A_Sugar_Toast), like as shown:
Screenshot from 2022-01-05 15-21-53

### CODE REVIEW

**Comment by S-kus** [comment]
@chimosky Please review it. If any changes are required, I will fix them.

**Comment by chimosky** [comment]
Reviewed ccea0d7, few comments;

- [ ] You're supposed to add the `On Trisquel` option as part of the ways to get sugar from the `Get Sugar` drop down menu.
- [ ] The box that contains the Trisquel Sugar page isn't of the same sizes as the other boxes, make it same size it should be 
aligned too.

This is what it looks like on my end;
Screenshot from 2022-01-05 13-39-51

**Comment by S-kus** [comment]
> * You're supposed to add the `On Trisquel` option as part of the ways to get sugar from the `Get Sugar` drop down menu.

Okay Sorry, I misunderstood by mistake. I can do that. Please confirm one more thing, does the SOAS link in get Sugar drop-down menu should also lead to [this page](https://wiki.sugarlabs.org/go/Sugar_on_a_Stick) than [installation page](https://wiki.sugarlabs.org/go/Sugar_on_a_Stick/Installation).




> * The box that contains the Trisquel Sugar page isn't of the same sizes as the other boxes, make it same size it should be
>   aligned too.
I fixed it, Now it will look Like this:
Screenshot from 2022-01-05 20-34-14

So, @chimosky please confirm, Do you want 1-drop down way Trisquel or 2-in the Get Sugar section. I will commit according to that.

**Comment by S-kus** [comment]
I added Trisquel in drop-down and removed it from this frontend section as required earlier. If you want it in the section again (I fixed the block size according to others) I will add that too.

**Comment by chimosky** [comment]
> Okay Sorry, I misunderstood by mistake. I can do that. Please confirm one more thing, does the SOAS link in get Sugar drop-down menu should also lead to [this page](https://wiki.sugarlabs.org/go/Sugar_on_a_Stick) than [installation page](https://wiki.sugarlabs.org/go/Sugar_on_a_Stick/Installation).

Yes it should lead to the first link.

> I fixed it, Now it will look Like this: Screenshot from 2022-01-05 23-14-23
> 
> So, @chimosky please review.

Reviewed, not tested.

From the screenshot you shared, it'll be nice if _On Trisquel_ is part of the software part of the drop-down, you've added it to the _XO_ and _RPi_ section while both are hardwares and Trisquel isn't.  It should in the same section as _SOAS_ as they're both bootable images.

**Comment by S-kus** [comment]
@chimosky I corrected the **_On Trisqel_** place. As shown:
Screenshot from 2022-01-07 08-16-30

**Comment by chimosky** [comment]
Apologies for making you do something extra, It'll be better if it was _TOAST_ instead of _On Trisquel_.

**Comment by S-kus** [comment]
@chimosky I changed it to TOAST from On Trisquel, as shown:

Screenshot from 2022-01-08 12-46-03

**Comment by chimosky** [comment]
> I added Trisquel in drop-down and removed it from this frontend section as required earlier. If you want it in the section again (I fixed the block size according to others) I will add that too.

Just reading the second part of your comment - apologies -, yes I think it should be added to the frontend section.
@quozl what'd you think?

**Comment by S-kus** [comment]
Okay @chimosky Once it gets confirmed, I will commit again for that part.

**Comment by quozl** [comment]
> @quozl what'd you think?

To fit Trisquel in to this layout, the OLPC XO could be removed, as it was last manufactured in 2017 and is aging in the field.  Still thousands active according to telemetry, but the potential market size for SoaS and Trisquel TOAST is larger.  Or you could combine SoaS and Trisquel and Sugar Live Build?

**Comment by S-kus** [comment]
@chimosky Could you please confirm once again, Do I need to replace OLPC XO with Trisquel or add a new block for it or anything else. I will do accordingly.

**Comment by chimosky** [comment]
> > @quozl what'd you think?
> 
> To fit Trisquel in to this layout, the OLPC XO could be removed, as it was last manufactured in 2017 and is aging in the field. Still thousands active according to telemetry, but the potential market size for SoaS and Trisquel TOAST is larger. Or you could combine SoaS and Trisquel and Sugar Live Build?

I agree with replacing the OLPC XO with Trisquel TOAST, I don't think combining all would be good.

**Comment by S-kus** [comment]
Updated view:
Screenshot from 2022-01-11 14-20-38
@chimosky Please review it.  If any changes are required, I will fix them.

File: index.html [hunk 1]
```diff
@@ -188,17 +188,18 @@ <h3 class="customHeaderStyle">Sugarizer</h3>
...
-                    an OLPC XO laptop.</p>
-                    <ul >
-                        <li class="listFont">
-                        <a href="http://wiki.laptop.org/go/Release_notes/13.2.9" class="hrefCustomColor">
-                            How to get Sugar on an OLPC XO Laptop</a></li>
-                    </ul>
+                    <p>The full Sugar environment
+                        installed on Trisquel GNU/Linux computer.</p>
+                        <ul >
+                            <li class="listFont"> Trisquel on a Sugar Toast a live/installable iso image that can be run from
```
Comment (chimosky) [inline index.html:198]: Should read _Trisquel on a Sugar Toast is a live/installable_.

Comment (S-kus) [reply index.html:198]: @chimosky Could you please confirm? Do I need to remove lines 195-198, and there is only one line `<p>Trisquel on a Sugar Toast is a live/installable.</p>`?

Comment (chimosky) [reply index.html:198]: I'm saying just that part of the text needs to change, every other part is okay.

**Comment by quozl** [comment]
Reviewed.  I'll make that change and get this rebased and merged.

**Comment by S-kus** [comment]
> Reviewed. I'll make that change and get this rebased and merged.

Thank you @quozl. 
@chimosky Thank you for this issue.

**Pull request merged**